### PR TITLE
Dashboard with Statistic

### PR DESCRIPTION
**Pull request by hungbkdn**

### CODE REVIEW

File: app/Http/Controllers/DashboardController.php [hunk 1]
```diff
@@ -3,16 +3,73 @@
...
+    }
+
+    /**
+     * Get total of new order, register, ... on day
+     *
+     * @param string $modelType type of model
+     *
+     * @return int
+     */
+    public function getTotalOf($modelType)
```
Comment (ghost) [inline app/Http/Controllers/DashboardController.php]: You can create a `trait` call `OrderStatistic` and create a function `getTotalOnDate`. To make flexible you can use a param for date. `getTotalOnDate($date = $curentDate)` by default $date is curent date, but when you want to get co another date, you can change param `$date`;

**Review by ghost — COMMENTED** [review]
Please check my comment

**Comment by hungbkdn** [comment]
I have updated, please check it again!
Thank you so much @at-vietpham

**Comment by ghost** [comment]
LGTM

**Pull request merged**